airflow/config_templates/default_airflow.cfg

```
@@ -125,6 +125,10 @@ dags_are_paused_at_creation = True
 # whose size is guided by this config element
 non_pooled_task_slot_count = 128
 
+# When not using pools, the number of backfill tasks per backfill
+# is limited by this config element
+non_pooled_backfill_task_slot_count = 128
```

is there a reason not to reuse the non_pooled_task_slot_count  config?

---

Usually people don't run backfills on the machine where scheduler runs. The backfill machine can be beefier than the scheduler machine. Thus I think it make sense to allow a larger task_slot_count for backfills and that is the reason why I make it another config.

---

> Usually people don't run backfills on the machine where scheduler runs

That is a very bold, unsubstantiated claim. And one that I disagree with.

---

Additionally if we want this to be the "same" as the setting above even when that changes we should do this 
```suggestion
non_pooled_backfill_task_slot_count = %(non_pooled_task_slot_count)s
```
And the config parser will set it to the same value.

---

@ashb Let me rephrase it a bit. In some cases, people do not run backfills on where scheduler runs...

Anyway I think we should still add this config so that people can fine tune the slot count if they launch backfill job on a machine other than scheduler